docs/usage.md

```diff
@@ -3,130 +3,441 @@
+
+2. **Correlate Records**:
+   - Specify the records you want to correlate. For example:
+
+    >*Correlate the newly detected IP [IP-ADDRESS] with the [INCIDENT-ID]*
+
+   - The assistant identifies and establish correlations between the specified records.
+
+3. **Review Linked Records**:
+   - Review the links and correlations to ensure they are correct and relevant. Verify that the connections between records provide meaningful insights.
+
+### Review Linked Records
+
+- Verify the accuracy and relevance of the linked records. Ensure that the connections provide a clear understanding of the relationships and contribute to effective analysis.
+
+### Integrate Linked Records into Workflow
+
+- Utilize the linked records to enhance your workflow. The connections between records can provide valuable insights and help streamline data analysis and response actions.
+
+### Conclusion
+
+Linking records is crucial for establishing connections between related pieces of information within FortiSOAR. By following the outlined steps and using the FortiAI assistant’s capabilities, users can effectively correlate records to improve their analysis and response efforts.
+
+>[!TIP]
+>When linking records, ensure that the connections are relevant and meaningful. Properly linking records can provide a more comprehensive view and improve the effectiveness of your analysis and response.
+
+## Create Record
+
+Creating records within the FortiAI assistant involves generating new entries for specified types, pages, or modules. This functionality allows users to add new data to the system, such as creating a new alert or incident record, to manage and track information effectively.
+
+>[!NOTE]
+>Ensure that you provide all the relevant details for records to be accurate and complete. Proper record creation supports effective data management and enhances the overall workflow.
+
+### Use Case Example: Creating a New Incident
+
+A common use case is creating a new incident record when a new threat is detected. For example, if an alert indicates a potential security breach, a new incident record can be created to track and manage the response.
+
+### Creating Records Using FortiAI Assistant
+
+1. **Initiate Record Creation**:
+   - Issue a command to create a record for the specified type, page, or module. For example:
+
+    >*Create a new incident record in the Incidents module*
+
+   - The assistant prompts you to provide the necessary details for the new record.
+
+2. **Specify Record Details**:
+   - Provide the required information for the record being created. For example:
+
+    >*Create a new alert with the following details: `[Details of the alert]`*
+
+   - The assistant generates the new record based on the provided details. If any mandatory details are missed, the bot requests for the missing information.
+
+3. **Review and Confirm**:
+   - Review the details of the newly created record to ensure accuracy. Confirm the creation to finalize the record entry.
```

Comment:
Confirm the creation to finalize the record entry. --> this is not needed but you can add that created record navigation link was presented to the user for easy navigation as part of response.

Reply:
For consistency